core/client/mutations/update-wishlist.ts

```diff
@@ -0,0 +1,43 @@
+import { getSessionCustomerId } from '~/auth';
```

Comment:
We no longer define mutation in this folder unless shared across multiple instances. Seems to me like you can simply use as needed in your form action.

Reply:
I plan to create small pr with refactoring for all wishlist mutations in nearest time